doc/nrf/known_issues.rst

```diff
@@ -2750,17 +2762,17 @@ SoftDevice Controller
 The issues in this section are related to :ref:`nrfxlib:softdevice_controller`.
 In addition to the known issues listed here, see also :ref:`softdevice_controller_limitations` for permanent limitations.
 
-.. rst-class:: v2-3-0
+.. rst-class:: v2-4-0 v2-3-0
 
 DRGN-18424: Rare assert when stopping the Periodic Advertiser role
   In rare cases, when a Periodic Advertiser instance is being stopped while another Periodic Advertiser instance is still running, the controller can assert.
 
-.. rst-class:: v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0
+.. rst-class:: v2-4-0 v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0
 
 DRGN-18424: Rare assert when stopping the Periodic Sync role
   In rare cases, when a Periodic Sync instance is being stopped while another Periodic Sync instance is still running, the controller can assert.
 
-.. rst-class:: v2-3-0
+.. rst-class:: v2-4-0 v2-3-0
```

Comment:
This line does not need the change, since DRGN-18651 has been fixed already in NCS 2.4.